Core/include/Acts/Seeding/SeedFilter.hpp

```diff
@@ -26,6 +26,12 @@ struct SeedFilterState {
   float zOrigin = 0;
   // number of minimum top SPs in seed confirmation
   size_t nTopSeedConf = 0;
+  // z minimum and maximum of middle component of the seed used to define the
+  // region of the detector for seed confirmation
+  float zMinSeedConf =
+      std::numeric_limits<float>::min();  // Acts::UnitConstants::mm
```

Comment:
This is a `0`. Is the z-range defined only in the `[0, inf)` range? Or did you mean to use `std::numeric_limits<float>::lowest()`?

Reply:
I think this would actually be the smallest positive number greater than 0

Reply:
that is indeed `lowest`, thank you for noticing